Formula/c/chkbit.rb

```diff
@@ -0,0 +1,39 @@
+class Chkbit < Formula
+  include Language::Python::Virtualenv
+
+  desc "Check your files for data corruption"
+  homepage "https://github.com/laktak/chkbit-py"
+  url "https://github.com/laktak/chkbit-py/archive/refs/tags/v4.2.1.tar.gz"
```

Comment:
looks like this is not on pypi? (can you switch the source to pypi one, we'd prefer pypi over github tarball)

<img width="1198" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/1580956/bdbcc88a-a862-47a6-9232-4149eb8c27e1">

Reply:
No problem, I've release it on pypi and switched the source.